### DISCUSSION TITLE
Node install, missing a piece?

### DISCUSSION THREAD

**Discussion by macasas** [saadeghi/daisyui#3032] [Q&A]
Hello everyone. I'm trying to add daisyui to a node project that is using the mustache template engine and htmx to render pages.

When I include the CDN into the head I get all the styles, but the daisyui installation guidelines say not to use CDN in production, so I am trying to install it as a tailwind plugin. [DaisyUI installation docs](https://daisyui.com/docs/install/)

I have installed tailwindcss and daisyUI

```
npm install -D tailwindcss 
npx tailwindcss init
npm i -D daisyui@latest`
```
and get
```
"daisyui": "^4.10.5",
"tailwindcss": "^3.4.3"
```
I have edited the tailwind config file

```
/** @type {import('tailwindcss').Config} */
import daisyui from "daisyui"

export default {
  content: [
    "./**/*.{html,js}",
    "./server/**/*.{html,js}",
    "./server/public/**/*.{html,js}",
    "./server/public/views/**/*.{html,js}"
  ],
  theme: {
    extend: {},
  },
  plugins: [
    '@tailwindcss/typography',
    daisyui,
  ],
},
}
```
And as you can see I have tried a few variations of the content path, all the way into my template html files. As I am using HTMX I don't have an output folder, templates are built on the fly and this is the only folder with html in it. All these paths have been tried on their own and in combination. Nothing changes.

I have run

`npx tailwindcss -i ./public/assets/styles/input.css -o ./public/assets/styles/tailwind.css --watch`
and i have the tailwind.css style linked into my html header.

When I remove the tailwind CDN link and just load the local generated tailwind.css file I get no daisy styles. Having tried lots of variations, I have to admit defeat and acknowledge I don't understand how this is supposed to work.

Can anyone suggest what I am doing wrong and how to connect the dots to have daisyui components? Thanks

**Comment by saadeghi** [comment]
Can you give us a minimum reproduction link please?

**Reply by macasas** [reply]
Sorry, I don't have a minimum reproduction repo.

**Reply by saadeghi** [reply]
Well, I don't see the issue when I run this config...
